src/components/homePage/Testimonials.vue

```diff
@@ -1,19 +1,238 @@
 <template>
-<div id="testimonials">
-	<h2>Testimonials</h2>
-	<!-- SIDE SCROLLER -->
-</div>
+  <div id="testimonials">
+    <div>
+      <img src="../../assets/testimonial_building.svg" id="building" />
+      <vueper-slides 
+        class="no-shadow" 
+        :bullets="false" 
+        :touchable="false" 
+        :disableArrowsOnEdges="true"
+        :arrowsOutside="true"
+        autoplay
+      >
+        <vueper-slide 
+          v-for="(slide, i) in information['slides']" 
+          :key="i"
+        >
+          <template v-slot:content>
+            <h2 id="hear">Hear what people have to say about CUTC!</h2>
+            <div class="vueperslide__content-wrapper" style="flex-direction: row">
+              <div id="slides">
+                <div id="title">
+                  <img :src="getImage(slide.image)" id="person" />
+                  <br />
+                  <strong> {{ slide.name }} </strong>
+                  <br/> 
+                  <strong> {{ slide.personDescription }} </strong>
+                </div>
+                <div id="content">{{ slide.content }}</div>
+              </div>
+            </div>
+          </template>
+        </vueper-slide>
+      </vueper-slides>
+    </div>
+    <br style="clear:both" />
+  </div>
 </template>
 
 <script>
+import { VueperSlides, VueperSlide } from 'vueperslides';
+import 'vueperslides/dist/vueperslides.css';
+import TestimonialInfo from "../../content/testimonials.json";
+
 export default {
 	name: 'Testimonials',
+	components: { VueperSlides, VueperSlide },
+  data () {
+    return {
+      information: TestimonialInfo
+    };
+  },
+  methods: {
+    getImage(pic) {
+      try {
+        return require("../../assets/" + pic);
+      } catch(e) {
+        throw Error(`testimonials.json references image that does not exist in assets: "${pic}"`);
```

Comment:
Would we want to show this error when we can't find a picture? Maybe having the `alt` attribute in `image` would be a good idea

Reply:
Yeah, this was used for testing. I agree with Adib: keep the catch, comment out the error throw, and add a dynamic alt tag!